src/Module.ts

```diff
@@ -390,6 +392,23 @@ export default class Module {
 		this.ast!.bind();
 	}
 
+	cacheInfoGetters(): void {
+		cacheObjectGetters(this.info, [
+			'dynamicallyImportedIdResolutions',
+			'dynamicallyImportedIds',
+			'dynamicImporters',
+			'exportedBindings',
+			'exports',
+			'hasDefaultExport',
+			'implicitlyLoadedAfterOneOf',
+			'implicitlyLoadedBefore',
+			'importedIdResolutions',
+			'importedIds',
+			'importers',
+			'isIncluded'
```

Comment:
While it is fine to cache most of these properties after `generateModuleGraph`, `isIncluded` must not be cached at this point. The value is only known after the end of the build phase when tree-shaking is done. A simple fix could be to just exclude it from this list, though, as the getter is not very expensive.